addons/hr/views/res_users.xml

```diff
@@ -72,11 +72,10 @@
                     </h5>
                 </xpath>
                 <xpath expr="//group[@name='other_preferences']" position="inside">
-                    <field name="is_hr_user" invisible="1"/>
-                    <field name="pin" string="Attendance PIN" readonly="not is_hr_user"/>
+                    <field name="pin" string="Attendance PIN"/>
```

Comment:
I get that we want to make this field no longer readonly, but is it a good idea to let `ANYONE` change a PIN. 
=> More of a security concern than anything else